### ISSUE TITLE
Share feature: Specify password errors.

### ISSUE THREAD

**Issue by gbonaspetti** [tracim/tracim#2226]
## Bug description & symptom

Iif I put a password less than 6 letters long or longer than 512 letters i have an error (not translated into French)

```Attention```
```Error while creating new share link```

It is apparently a 400 on the length of the password that does not respect the spec.

**Comment by inkhey** [comment]
In both case, it give error 2001 about validation. There is no distinct error code in backend for such case yet (and it's unclear how to do so), so you can't currently rely on error code to specify this.

**Comment by PhilippeAccorsi** [comment]
I think for this release we can use this fix and notify user about necessary lenght of password if lenght is not good.

**Comment by PhilippeAccorsi** [comment]
Its not really working. If I choice a wrong password (not between 6 and 512 char) this flash message appear
~~~
Attention
Le mot de passe doit être comprise entre 6 et 512 caractères et le(s) email(s) doit (doivent) être valide(s)
~~~

Erreur en français : "comprise" => "compris"

~~~
Warning
The password length must be between 6 and 512 characters and the email(s) must be valid
~~~

and the page is refreshed. This is not user freindly if I need to click again on "new" and try again to choice a password. In this case flash message do not start refresh of page.

I don't also understand why we seen information about "and the email(s) must be valid", if email is not valid a specific message is show and no relation exist between password and email.

**Comment by PhilippeAccorsi** [comment]
Also I think we need to use same flash message about invalid email like we have when we try to create a new user in tracim.

**Comment by PhilippeAccorsi** [comment]
I don't remember if after discuss we have decided to add specific control of email format (like we use in form create user).

**Comment by PhilippeAccorsi** [comment]
This issue is verified. all comment are solved in tracim 2.4.0 with other issue.
Verified.
